### PR TITLE
Some housekeeping in OCA/maintainer-tools

### PR DESCRIPTION
**Pull request by sbidoul**
The list of commits should explain what I'm doing here

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,76 @@
+import os
```
Comment (elicoidal) [inline setup.py]: I would suggest to take the opportunity to add headers for copyright and license to all files.

**Comment by sbidoul** [comment]
License headers added.

**Comment by sbidoul** [comment]
No objection if I merge this?

**Comment by pedrobaeza** [comment]
OK, go ahead

**Pull request merged**